pkgs/poetry/update

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -i bash -p curl nix-prefetch-github jq
+#! nix-shell update-poetry-shell.nix -i bash
 
+set -x
```

Comment:
I've kept it since the script is small and that can be helpful anyway. I can remove the print if needed